roles/bootstrap/templates/bootstrap-ci-network-nm-connection.nmconnection.j2

```diff
@@ -10,17 +10,33 @@ mac-address={{ iface_info.mac | trim | lower }}
 mtu={{ iface_info.mtu }}
 
 [ipv4]
+{% if iface_info.ip is not defined -%}
+method=disabled
+{% else -%}
 method=manual
 addresses={{ iface_info.ip }}
+never-default=true
```

Comment:
This is from commit 78f8cad13041c9c74e11acd7707970c954f0d7e0 - in main. Accidentally included when I pushed the MR against staging I assume.